main.tf

```diff
@@ -57,7 +57,7 @@ module "google_mysql_db" {
   create_timeout      = var.db_timeout
   update_timeout      = var.db_timeout
   delete_timeout      = var.db_timeout
-  user_name           = var.user_name
+  user_name           = var.default_user_name
```

Comment:
can we call it root user instead? that's the normal nomenclature. this is a root user and the others are additional users.

Reply:
like-

`root_user_name`
`root_user_password`
`root_user_host`

Reply:
Agreed. The Google submodule we used however calls it default - so kept it synonymous with that.

Let me verify if this user indeed gets root privileges assigned to it or not. We can call it root then.

Reply:
Hi @dibosh I have made this change.